app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -238,4 +243,10 @@ case class ServiceDetails(serviceName: String, serviceUrl: String)
 
 case class RawDocumentationContent(contentType: String, content: String)
 
-case class ErrorResponse(code: Option[String] = None, message: Option[String] = None)
\ No newline at end of file
+case class ErrorResponse(code: Option[String] = None, message: Option[String] = None)
+
+object DocsVisibility extends Enumeration {
```

Comment:
Scala native Enumerations has issues - consider alternative like Enumeratum. See TPDF work that I did to change Enumerations.

https://github.com/hmrc/third-party-developer-frontend/commit/b4a1cfbfd28b35b19d0befa2a9f69aff60ce8157

Reply:
Love to, should do as part of a larger piece of work though?